common/artifacts/901_wc_artifacts_weapon_onehanded.txt

```diff
@@ -98,4 +124,63 @@ serathil = {
 
 	picture = "GFX_serathil"
 	slot = weapon
-}
\ No newline at end of file
+}
+the_bladefist = {
+	quality = 3
+	flags = { combat_weapon sword }
+	
+	stacking = no
+	
+	active = {
+		trait = one_handed
+		shared_physical_weapon_trigger = yes
+	}
+	
+	martial = 1
+	combat_rating = 16
+	
+	monthly_character_prestige = 0.15 
+
+	command_modifier = {
+		morale_offence = 0.06
+	}
+
+	picture = "GFX_the_bladefist"
+	
+	slot = weapon 
+
+}
+bloodsurge = {
+	quality = 3
+	flags = { combat_weapon sword }
+
+	active = {
+		is_adult = yes
+
+		is_magic_class_trigger = yes
+	}
+
+	stacking = no
+	indestructible = no
+
+	learning = 2
+	combat_rating = 10
+
+	monthly_character_prestige = 0.3
+	monthly_character_piety = 0.3
+
+	death_god_opinion = 15 #+10 Opinion to death god believers
+	general_opinion -5
+
+	picture = "GFX_bloodsurge"
+	slot = weapon
+}
```

Comment:
Why create a sword that no non-magic-user can wield? Plus, I'd either limit it to death_god_religion owner or get rid of the death_god_opinion.

Reply:
Bloodsurge was a one-handed weapon gives intelligence so limiting it to magic guys would be better. and i will  get rid of opinions on all artifacts

Reply:
I guess I can already predict Zumb won't like this... 😅